scipy/special/cephes/hyp2f1.c

```diff
@@ -590,3 +601,30 @@ static double hyp2f1ra(double a, double b, double c, double x,
 
     return f0;
 }
+
+
+/*
+    15.4.2 Abramowitz & Stegun.
+*/
+static double hyp2f1_neg_c_equal_bc(double a, double b, double x)
+{
+    /*
+    	"converting floating point to integers is shockingly expensive"
+	
+	Therefore, we stay with double the entire time.
+    */
+    double k;
+    double sum = 0;
+    double x_k = 1; // x^k
+    double k_f = 1; // k!    
+
+    for (k = 0; k <= -b; k++) {
```

Comment:
Note that this is going to involve summing a very large series which could be inaccurate (when there's cancellation). Probably need to investigate when it will give bad results and return `nan` in those cases.

Reply:
No. The series is only summed to 100 at most.

Reply:
However, the statement that the different definition of the `c=b=negint` case gives the same result for `b<-100` is questionable, consider e.g. `hyp2f1(10, -101, -101, 0.9)`